src/main/java/de/tum/in/www1/artemis/web/rest/dto/pageablesearch/PageableSearchDTO.java

```diff
@@ -3,15 +3,13 @@
 import de.tum.in.www1.artemis.domain.enumeration.SortingOrder;
 import de.tum.in.www1.artemis.web.rest.dto.SearchResultPageDTO;
 
-// TODO (follow-up, as it changes 40+ files) rename this, move & rename PageableSearchDTO, move UserPageableSearchDTO.
-
 /**
  * Wrapper for a generic search for any list of entities. The result should be paged,
  * meaning that it only contains a predefined number of elements in order to not fetch and return too many.
  *
  * @see SearchResultPageDTO
  */
-public class BasePageableSearchDTO<T> {
+public class PageableSearchDTO<T> {
```

Comment:
Are we using this directly as a class? Or would it make sense to make this an abstract class?

Reply:
You are right, it is not used at the moment. However, I would keep it as-is because in the future it could be utilized and making it abstract would be more confusing than beneficial in my opinion. What do you think?

Reply:
I'd agree with @rstief. From my point of view, there is no obvious reason we should prevent developers from performing a "search" operation that simply sorts a pageable list.